inz-demo/internetnz/Issues/nethuiTicket/description.xml

```diff
@@ -0,0 +1,83 @@
+<IssueSpecification version="4">
+	<Name>
+		<en>Demo INZ Nethui Access Pass</en>
+		<nl>Demo INZ Nethui Access Pass</nl>
+	</Name>
+	<ShortName>
+		<en>Demo INZ Nethui Access Pass</en>
+		<nl></nl>
+	</ShortName>
+	<SchemeManager>inz-demo</SchemeManager>
+	<IssuerID>internetnz</IssuerID>
+	<CredentialID>nethuiTicket</CredentialID>
```

Comment:
nethuiAccessPass

Reply:
Fair enough. I changed here and at demo UI.